integration-tests/typescript-koa/src/petstore-expanded.yml/generated.ts

```diff
@@ -67,8 +79,24 @@ export function bootstrap(
     }
 
     const { status, body } = await implementation.findPets(input, ctx)
+
+    ctx.body = responseValidationFactory(
+      [
+        [
+          "200",
+          z.array(
```

Comment:
Perhaps it's worth splitting these out to a `schemas.ts`

Reply:
Mostly done in [8aed680](https://github.com/mnahkies/openapi-code-generator/pull/29/commits/8aed680b5b4e9d7bb6b9448025662f5459ccb7ee)